services/app-web/src/pages/StateSubmission/RateDetails/SingleRateCert/SingleRateCert.tsx

```diff
@@ -164,6 +164,7 @@ export const SingleRateCert = ({
                     label="Upload one rate certification document"
                     renderMode="list"
                     aria-required
+                    allowMultipleUploads={false}
                     error={showFieldErrors('rateDocuments')}
```

Comment:
I'm getting the error `You must upload a rate certification` when viewing a unlocked submission with multiple rate cert files.

Reproduction steps:
- deploy app using `main` branch.
- create a submission with multiple rate cert files and submit.
- shut down the app, except for the Postgres instance.
- switch to this branch and only deploy all but Postgres using this command `./dev local --api --web --s3 --otel`
- you may have to manually start the Postgres instance in docker.
- unlock the submission we made using `main`. (This simulates unlocking an existing submission)
- as a state user view the rate details page and hit continue. 
- see error message not display correct text.